src/examples/authentication_with_jobs.yml

```diff
@@ -0,0 +1,30 @@
+description: >
```

Comment:
I think maybe a potentially better example of job usage could be showing how you could use workspaces to let's say create an Android and IOS app, and then upload both artifacts at the end of the workflow in a copy job.